dashboard/cookbook/jstranslation.md

```diff
@@ -0,0 +1,229 @@
+
+There are many ingenious ways in which webpages encode content, and the proxy has various levels of support for all these schemes (usually involving a combination of features). When in doubt, feel free to contact support for advice!
+
+## Marking Resources
+
+You can mark a resource as translatable manually on the Resource screen or  using a prefix in Advanced Settings.
+
+### Manual
+
+All collected resources are listed in Discovery > Resources and Content > Resources. All "pages" and files that are not of the `text/html` content type will go here.
+
+What sets resources apart from pages is that by default, they have no associated *source entries* or translations. Marking a resource as translatable means declaring that it does have translatable text that can be stored as source entries (and accessed via the Workbench).
+
+So, to mark a resource as translatable:
+
+![Switch to List View](/img/dashboard/resources_list_view_filter.png)
+
+1. switch from thumbnail view to list view
+
+2. click on the "Translate" button in the hover menu of a resource
+
+![Translate Resource](/img/dashboard/resources_list_view_translate.png)
+
+The resource is moved to the page list and from that point onward, it can be opened on the Workbench. Note, however, that we have not yet told the proxy what and how to translate in it.
+
+### Prefix
+
+You can do the marking via prefixes. Go to Advanced settings to find the "Mark Multiple Resources as Translatable" text field. Copy & paste the prefixes of your choice and click on Save to apply your changes.
+
+![Mark Multiple Resources](/img/dashboard/advanced_multiple_resources.png)
+
+Note that in the screenshot above, HTTP and HTTPS prefixes are handled separately, a recommended practice for sites that support both kinds of traffic. Prefixes are treated as simple strings by the proxy when matching them against a pathname. You are free to add as many of them as you like.
+
+This feature is made available because cherry-picking resources for translation is not always feasible. For instance, versioned URLs are liable to create new resources on a project whenever a file is updated on the original site (the proxy keeps these URLs separate by default), but the new resources are not marked automatically.
+
+You will recognize those cases where you want to apply the exact same translation rules and process to a set of URLs that differ in minimal ways. A resource prefix will let you do do this without having to mark things one-by-one as they come.
+
+## Annotating JS/JSON
+
+Picking up JSON/JS/XML content *wholesale* would be both costly and unwieldy, however. When you have successfully identified the source of content and earmarked it for translation, the last major task is to annotate those parts of it that really want to translate. **JS/JSON paths** and **Xpaths** can be used for this purpose.
```

Comment:
@lsoos Is this really good grammatically, that after `however` there is a dot and a start of a new sentence?